new/LIT7411Emdengel.xml

```diff
@@ -0,0 +1,77 @@
+                <title xml:lang="en" corresp="#t1">From virgin, without a man, is Nativity; our Saviour Christ without woman from the Father has 
+                    been born, as before…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The author of the qǝne is not indicated in the manuscript. The type of the qǝne was indicated as 
+                    <foreign xml:lang="gez">ሥላሴ፡</foreign>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Sellase"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-04-29">Created entity</change>
+            <change when="2025-04-29" who="CH">Minor updates for uniformity to other Bm records</change>
+            <change when="2025-04-30" who="CH">Corrections and update of title according to the review of Denis Nosnitsin</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="PetermannIINachtrag24"/>, fol. 7ra. Division into verses is indicated 
+                    with ። .</note>
+                <ab>
+                    ሥላሴ፡
+                    <l n="1">እምድንግል፡ እንበለ፡ ብዕሲ፡ ልደተ፡ ክርስቶስ፡ መድኃኒነ፡ እንበለ፡ 
+                        ብ<supplied reason="omitted" resp="CH">እ</supplied>ሲት፡ እምአብ፡ ከመ፡ ቀዳሚ፡ ተወልደ።</l> 
+                    <l n="2">ስመ፡ ወልድናሁ፡ ለኵሉ፡ አግሀደ።</l>
+                    <l n="3">ኢተቶስሐ፡ በስሙ፡ እምስሞሙ፡ ወኢተዳውዓ፡ <sic>ህ<add place="above">ቀ</add>ደ።</sic></l>
```

Comment:
Could it be "ወኢተጸውዓ"? I'm not sure

Reply:
I though of it, but it is difficult since the next word is also unclear.

Reply:
Yes, indeed